src/models/TerraformData.js

```diff
@@ -0,0 +1,97 @@
+import {
+  DefaultData,
+} from 'leto-modelizer-plugin-core';
+
+class TerraformData extends DefaultData {
+  /**
+   * Get the value of a variable.
+   * @param {string} variable - Variable to get value.
+   * @returns {string} Value of the variable.
+   */
+  getVariableValue(variable) {
+    const args = variable.split('.');
+    if (args.length === 2) {
```

Comment:
length can't be >2 ?

Reply:
No, the maximum possible args in a variable reference is 2; otherwise it will be an invalid reference.